libraries/lib-concurrency/ParallelProgress.h

```diff
@@ -0,0 +1,55 @@
+/**********************************************************************
+ 
+ Audacity: A Digital Audio Editor
+ 
+ @file ParallelProgress.h
+ 
+ Paul Licameli
+ 
+ **********************************************************************/
+#ifndef __AUDACITY_PARALLEL_PROGRESS__
+#define __AUDACITY_PARALLEL_PROGRESS__
+
+#include <atomic>
+
+namespace Parallel {
```

Comment:
What is the justification of the capital P here?

Reply:
None really.

We don't have very well defined conventions.

We have already namespace Observer